test/functional/update/newest-ghosted/test.bats

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bats
+
+load "../../swupdlib"
+
+os_release_file=1c1efd10467cbc1dddd8e63c3ef4d9099f36418ed5372d080a1bf0f03a49ab05
+
+setup() {
+  clean_test_dir
+  create_manifest_tar 10 MoM
+  sign_manifest_mom 10
+  create_manifest_tar 10 os-core
+  create_manifest_tar 10 test-bundle1
```

Comment:
Do you need to include build 10 for this test? I see you're updating from build 20 to 30, and no new bundle includes are added for installed bundles in build 30, so the manifest content from build 10 would be irrelevant here. Unless I'm missing something :-)